### PR TITLE
10x28 changes

### PR DESCRIPTION
**Pull request by AmoryBlaine**
- Damage multipliers removed from most rifles
- M4RA, M56 Smartgun, Type 71, Type 71c, PKP all share the same ammo type, 10x28 with a higher base damage than standard ammo.
- UPP given unique 10x31 ammo that is the same stats wise as 10x28.
- Updates mapless M4RA variant icons to match M41A palette

### CODE REVIEW

File: code/modules/projectiles/magazines/rifles.dm [hunk 1]
```diff
@@ -138,59 +138,59 @@
...
+	desc = "A magazine of armor-piercing 10x28mm rounds for use in the M4RA battle rifle."
+	default_ammo = /datum/ammo/bullet/rifle/heavy/ap
+	max_rounds = 30
 	ammo_band_color = AMMO_BAND_COLOR_AP
 
 /obj/item/ammo_magazine/rifle/m4ra/ext
-	name = "\improper M4RA extended magazine (10x24mm)"
-	desc = "A magazine of armor-piercing 10x24mm rounds for use in the M4RA battle rifle. Holds an additional 10 rounds, up to 35."
+	name = "\improper M4RA extended magazine (10x28mm)"
+	desc = "A magazine of armor-piercing 10x28mm rounds for use in the M4RA battle rifle. Holds an additional 10 rounds, up to 35."
```
Comment (private-tristan) [inline code/modules/projectiles/magazines/rifles.dm:160]:
```suggestion
	desc = "A magazine of armor-piercing 10x28mm rounds for use in the M4RA battle rifle. Holds an additional 10 rounds, up to 40."
```

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by GriffinMan4455** [comment]
SG's feel fucking awesome, I'm loving this change, especially combined with the SG Belt PR, though I think it'd be nice if the locker had three drums in total, if only so that SG's have a choice of having an ammobearer for just-in-case, or forgoing it in order to have a more customizable loadout  
If this were to be merged alongside the sidearm overhaul I'd pog, because as it stands our sidearms are kinda like...eh? Revolver's fun for stun, and it helped me save the Corpsman, but otherwise they chew through a ton of ammo (which is a lot of loadout space) in exchange for not doing a lot, especially with how low SG's are on ammo now.
Sadly I can't speak for how this affects RFN, but from that op they looked fine, at least.

File: code/datums/ammo/bullet/rifle.dm [hunk 2]
```diff
@@ -91,47 +93,93 @@
...
+
+/datum/ammo/bullet/rifle/heavy/ap
+	name = "armor-piercing 10x28 bullet"
+	headshot_state = HEADSHOT_OVERLAY_MEDIUM
+	damage = 40
+	penetration = ARMOR_PENETRATION_TIER_8
+
+/datum/ammo/bullet/rifle/heavy/heap
+	headshot_state = HEADSHOT_OVERLAY_HEAVY
+	name = "armor-piercing 10x28 bullet"
```
Comment (private-tristan) [inline code/datums/ammo/bullet/rifle.dm:155]:
```suggestion
	name = "high explosive armor-piercing 10x28 bullet"
```

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by Doubleumc — CHANGES_REQUESTED** [review]
> Missing icon_state for /obj/item/attachable/uppsmartbarrel in 'icons/obj/items/weapons/guns/attachments/barrel.dmi'.

image
https://github.com/cmss13-devs/cmss13-pve/actions/runs/10650509523/job/29521976191?pr=376

File: code/modules/projectiles/guns/smartgun.dm [hunk 3]
```diff
@@ -761,3 +761,21 @@
...
+	flags_gun_features = GUN_WIELDED_FIRING_ONLY
+	icon = 'icons/obj/items/weapons/guns/guns_by_faction/upp.dmi'
+	icon_state = "dp88"
+	item_state = "dp88"
+	starting_attachment_types = list(/obj/item/attachable/uppsmartbarrel)
+
+/obj/item/weapon/gun/smartgun/rmc/Initialize(mapload, ...)
+	. = ..()
+	MD.iff_signal = FACTION_UPP
+ */
```
Comment (Doubleumc) [inline code/modules/projectiles/guns/smartgun.dm:781]:
```suggestion
```
Still need this commented out bit?

Comment (AmoryBlaine) [reply code/modules/projectiles/guns/smartgun.dm:781]: Only until I copypaste it into a notepad in five seconds

**Review by Doubleumc — APPROVED** [review]

**Pull request merged**